cypress-smoketests/integration/case-creation.spec.js

```diff
@@ -0,0 +1,121 @@
+const {
+  closeScannerSetupDialog,
+  goToCreateCase,
+  goToReviewCase,
+  saveCaseForLater,
+  serveCaseToIrs,
+} = require('../support/pages/create-paper-case');
+const {
+  completeWizardStep1,
+  completeWizardStep2,
+  completeWizardStep3,
+  completeWizardStep4,
+  filingTypes,
+  goToDashboard,
+  goToStartCreatePetition,
+  goToWizardStep1,
+  goToWizardStep2,
+  goToWizardStep3,
+  goToWizardStep4,
+  goToWizardStep5,
+  hasIrsNotice,
+  submitPetition,
+} = require('../support/pages/create-electronic-petition');
+const {
+  fillInCreateCaseFromPaperForm,
+} = require('../../cypress/support/pages/create-paper-petition');
+const { getUserToken, login } = require('../support/pages/login');
+const { goToMyDocumentQC } = require('../support/pages/document-qc');
+
+let token = null;
+
+describe('Petitioner', () => {
+  before(async () => {
+    const results = await getUserToken(
```

Comment:
@ericsorenson I think there's been some talk about figuring out how we can stop relying on these predetermined Cognito users. Do you know if there's a card in the backlog somewhere about this?

Reply:
We talked about this in our last launch planning meeting, and there's a card created for that in the backlog 👍